stats/includes/member.inc

```diff
@@ -607,7 +607,10 @@ function showMbrHistory($user, $tally_name)
         "<div style='display:flex;flex-direction:column;align-items:center'><div id='tallyboard_deltas' style='width:600px;height:300px'></div></div>"
     );
     $t->end();
+    $graphs = [
+        ["barChart", "tallyboard_deltas", $tallyboard_deltas_config],
+    ];
     echo "<script type='application/javascript'>";
-    echo "$(function(){barChart('tallyboard_deltas', " . json_encode($tallyboard_deltas_config) . ")});";
+    echo build_svg_graph_inits($graphs);
```

Comment:
Do we think this would be more readable in a single call?
```php
echo build_svg_graph_inits([["barChart", "tallyboard_deltas", $tallyboard_deltas_config]]);
```
Maybe not if the formatter smears it across multiple lines. Whatever you decide is fine.